app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,39 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(8, "bottom");
```

Comment:
I think you don't need to define margin here, because you're using the shared helper and it looks like it will default to returning the class for this much margin anyway?

Reply:
No longer using the shared helper